library_generation/utilities.sh

```diff
@@ -76,15 +76,15 @@ search_additional_protos() {
 
 # get gapic options from .yaml and .json files from proto_path.
 get_gapic_opts() {
-  gapic_config=$(find "${proto_path}" -type f -name "*gapic.yaml")
+  gapic_config=$(find "$proto_path" -type f -name "*gapic.yaml")
```

Comment:
prefer "${var}" over "$var"

https://google.github.io/styleguide/shellguide.html

Reply:
Thanks for the reference, I'll modify code accordingly.

Reply:
Thanks Tomo for sharing this! This is really helpful, I see a section regarding [when to use shell](https://google.github.io/styleguide/shellguide.html#when-to-use-shell) which I think it's worth discussing more in our next meeting.